### PR TITLE
WINUI-843320 Prepared the overview UG doument content for Shimmer control in WinUI.

### PR DESCRIPTION
**Pull request by GilbertmanojSF3971**
Added the Overview content for the SfShimmer control ug Documentation.

### CODE REVIEW

File: winui-toc.html [hunk 1]
```diff
@@ -531,6 +531,16 @@
        <li><a href="/winui/SegmentedControl/Disable-Items">Disable Items</a></li>
    </ul>
 </li>
+<li>
+    SfShimmer Control
+   <ul>
+       <li><a href="/winui/SfShimmerControl/Overview">Overview</a></li>
```
Comment (Ganesh-Mariappan) [inline winui-toc.html:537]: @GilbertmanojSF3971 Check and change the path properly. change the folder name to Shimmer instead of SfShimmer.

Comment (GilbertmanojSF3971) [reply winui-toc.html]: @Ganesh-Mariappan Changed the path and named the SfShimmer into Shimmer for the folder

File: winui/Shimmer/Overview.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: Overview of WinUI SfShimmer control | Syncfusion
```
Comment (Ganesh-Mariappan) [inline winui/Shimmer/Overview.md:3]: @GilbertmanojSF3971 title: Overview of WinUI Shimmer control | Syncfusion

Comment (GilbertmanojSF3971) [reply winui/Shimmer/Overview.md]: @Ganesh-Mariappan Changed the SfShimmer into Shimmer in the tiltle.

File: winui/Shimmer/Overview.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+platform: WinUI
+control: Shimmer
+documentation: ug
+---
+
+# WinUI Shimmer Overview
+
+The WinUI Shimmer control enhances the responsiveness of your application by displaying a shimmer effect during content loading or background processing.
+
+![WinUI Shimmer control overview](Shimmer_images/Overview_images/winui_shimmer_overview.gif)
```
Comment (Manivannan-E) [inline winui/Shimmer/Overview.md:14]: @GilbertmanojSF3971 Change the overview image.

File: winui/Shimmer/Overview.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+## Key Features
+
+* `Shimmer types`: Seven built-in shimmer types.
```
Comment (Manivannan-E) [inline winui/Shimmer/Overview.md:18]: @GilbertmanojSF3971 Don't use separate folder for every image.

File: winui/Shimmer/Overview.md (hunk 3 above)
Comment (GilbertmanojSF3971) [reply winui/Shimmer/Overview.md]: @Manivannan-E Sir Changed the Overview image.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**